source/_docs/load-and-performance-testing.md

```diff
@@ -1,200 +1,73 @@
+
+## Preparing for Tests
+The procedure for executing a load test and a performance test are similar:
+
+1. Enable [New Relic Pro](/docs/new-relic) within the Site Dashboard on Pantheon to ensure you have clear reporting to monitor response times.
+
+   * Set your [apdex](https://docs.newrelic.com/docs/apm/new-relic-apm/apdex/apdex-measuring-user-satisfaction#score) threshold according to your business rules (.5 is the default).  Be careful not to set this too high otherwise you will not get as many transaction traces in New Relic.  
+   * If you have any transactions that you want to ensure transaction trace is on be sure to set them up as [key transactions](https://docs.newrelic.com/docs/apm/transactions/key-transactions/key-transactions-tracking-important-transactions-or-events).
+
+2. Select a load testing tool:
+
+ * SaaS Solutions
+  * [Blazemeter](https://www.blazemeter.com)
+  * [Load Impact](https://loadimpact.com)
+ * Open Source tools
+  * [Jmeter](http://jmeter.apache.org)
+  * [Locust](http://locust.io/)
+
+  The Pantheon onboarding team uses Locust, an open source load testing tool, because it's fairly easy to build out test scripts and it allows you to crawl the site instead of using predefined URLs. Crawling the site has the added benefit of hitting every page that is linked to anywhere on the site.  This exposes edge case performance bottlenecks that would have gone undetected under more rigid tests.
+
+  Ultimately, is doesn't matter what tool you decide to use as long as it allows you to test your site properly.  Be sure to allow for any authenticated traffic as well as anonymous.  
+
+3. Determine how much load to apply for your test.
+
+  * **Performance Tests**: Smaller loads should suffice, as you should be able to see transactional bottlenecks with 10-20 concurrent users.
+  * **Load Tests**: Determine how many concurrent users the site is expected to serve based on historical analytics for the site. Identify the peak hourly sessions and average session duration, then do some math: `hourly_sessions / (60 / average_duration) = Concurrent Users`
+
+
+###Running the Tests
+If this is a performance test, be sure to run the test on a development environment (Dev or [Multidev](/docs/multidev)) without caching. Run load tests on the Live environment before launching the site. If the site is already launched, use the Test environment instead.
+<div markdown="1" class="alert alert-danger" role="alert">
+###Warning {.info}
+We do not recommend load testing on the Live environment if the site has already launched because you risk overwhelming your live site and causing downtime.
+</div>
+Note the start time for the test. As the test executes, it's a good idea to keep a close eye on [log files](/docs/logs). Make note of any errors and warnings that pop up during test to fix.

```

Comment:
Which log files should I watch? Do they differ based on the type of test being run?